core/src/main/scala/ai/h2o/sparkling/backend/utils/H2OContextExtensions.scala

```diff
@@ -105,4 +109,74 @@ trait H2OContextExtensions extends RestCommunication with RestApiUtils {
       throw new IllegalArgumentException(s"Supported LOG container is either LOG or ZIP, specified was: $logContainer")
     }
   }
+
+  protected def getAndVerifyWorkerNodes(conf: H2OConf): Array[NodeDesc] = {
```

Comment:
no

Reply:
I have just moved the methods here as they are now used in both backends.

Actually, we do not need to check for the web to be opened in case of internal H2O backend as we don't allow user to change it, but I would rather have consistent + we can be really sure the web is open.